server/src/main/scala/actors/ProdDrtParameters.scala

```diff
@@ -158,4 +160,6 @@ case class ProdDrtParameters @Inject()(config: Configuration) extends DrtParamet
   override val stalePredictionHours: FiniteDuration = config.get[Int]("crunch.stale-prediction-hours").hours
 
   override val enableStaffingPageWarnings: Boolean = config.get[Boolean]("feature-flags.enable-staffing-page-warnings")
+
+  override val codeShareExceptions: Set[String] = config.getOptional[String]("code-share.exceptions").map(_.split(",").toSet).getOrElse(Set())
```

Comment:
Do we need to worry about trimming and uppercasing the code-share.exceptions values when parsing ?

Reply:
Yeah good call. I've added trim & uppercasing